build.gradle

```diff
@@ -42,6 +42,9 @@ dependencies {
 
   // Sentry reporting
   compile "io.sentry:sentry-spring:1.7.30"
+
+  // test
+  testImplementation('org.mockito:mockito-junit-jupiter:2.23.0')
```

Comment:
Mockito is also included in the spring test starter, any reason we can't use that and not declare the dependency again?

Reply:
I thought it wasn't included, will remove it as well. :)